shopsafe-frontend/src/app/components/check-in-modal/check-in-modal.component.css

```diff
@@ -42,7 +42,7 @@
 }
 
 #check-in-modal #invalid-check-in {
-    background: #68BBCF;
+    background: #bdbdbd;
```

Comment:
instead of greying it out, you can try using the disable attribute instead -- if you need an example, look at the search-bar component html @Raulp8

Reply:
yeah i dont want the button to be disabled just because i have the invalid checkin button mapped to a function that briefly displays the red text so that users understand that they have to move all the sliders. I just felt like it might not be apparent that you have move the slider in order to submit with just a disabled button

Reply:
Okay, sounds good!